### PR TITLE
Fix package.json command name

### PR DESCRIPTION
**Pull request by tyler-cai-microsoft**
Fix to internal build pipeline PR: https://github.com/microsoft/FluidFramework/pull/15638

The package.json command should be with `:coverage` as the npm build pipeline add `:coverage` to the command

### CODE REVIEW

**Review by jikim-msft — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -33,10 +33,10 @@
 		"ci:test:mocha:coverage": "c8 --no-clean npm run test:mocha:report",
 		"ci:test:realsvc:local": "pnpm run -r --no-sort --stream --no-bail test:realsvc:local:report",
 		"ci:test:realsvc:local:coverage": "c8 --no-clean pnpm recursive --no-sort --stream --no-bail run test:realsvc:local:report",
-		"ci:test:realsvc:local:full": "pnpm run -r --no-sort --stream --no-bail test:realsvc:local:report:full",
+		"ci:test:realsvc:local:full:coverage": "c8 --no-clean pnpm recursive --no-sort --stream --no-bail run test:realsvc:local:report:full",
```
Comment (alexvy86) [inline package.json:36]: Was there a reason for `pnpm recursive ... run` instead of ` pnpm run -r ...` like before (or in the other changed line)?

**Review by alexvy86 — COMMENTED** [review]
Oh, also I think we'll want both versions of the scripts, with and without `:coverage`. The pipeline step that calls them is on an if-else block that depends on a variable we can change when kicking off a run (and defaults to "run the with-coverage" versions, IIRC). So we can technically request that it calls the script names without the `:coverage` suffix. That's why all the ones around these new ones come in both flavors.

File: package.json (hunk 1 above)
Comment (tyler-cai-microsoft) [reply package.json:36]: It's so it's like the other `local:coverage` command

**Comment by tyler-cai-microsoft** [comment]
> Oh, also I think we'll want both versions of the scripts, with and without `:coverage`. The pipeline step that calls them is on an if-else block that depends on a variable we can change when kicking off a run (and defaults to "run the with-coverage" versions, IIRC). So we can technically request that it calls the script names without the `:coverage` suffix. That's why all the ones around these new ones come in both flavors.

OK, for sake of fixing the current build pipeline, I'll add it as a separate PR.

**Pull request merged**